_api-reference/cluster-stats.md

```diff
@@ -31,7 +30,7 @@ All cluster stats parameters are optional.
 
 Parameter | Type | Description
 :--- | :--- | :---
-&lt;node-filters&gt; | List | A comma-separated list of [node filters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/nodes-apis/index/#node-filters) that OpenSearch uses to filter results.
+&lt;node-filters&gt; | List | A comma-separated list of [node filters]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/index/#node-filters) that OpenSearch uses to filter results.
```

Comment:
Capitalize title of linked page/section:

`[Node filters]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/index/#node-filters)`